src/v2/api/group/GroupService.ts

```diff
@@ -69,4 +78,40 @@ export class GroupService {
 
     return results;
   }
+
+  async addUnregistered(groupId: string, body: EmailDTO) {
+    await this.userRepository.addUnregistered(groupId, body);
```

Comment:
this logic is wrong. user repository should contain basic methods without any other logic except for access to database